src/window_message.cpp

```diff
@@ -126,7 +118,41 @@ void Window_Message::StartMessageProcessing() {
 			}
 		}
 	}
+}
+
+void Window_Message::StartMessageProcessing() {
+	contents->Clear();
+
+	if (Game_Message::is_word_wrapped) {
```

Comment:
[Copying from chat is someone is going to read the history in the future] "Player::ResetGameObjects already calls Game_Message::Init() which results in SemiClear being called."